include/z64save.h

```diff
@@ -54,6 +55,55 @@ typedef enum {
 #define MAGIC_NORMAL_METER 0x30
 #define MAGIC_DOUBLE_METER (2 * MAGIC_NORMAL_METER)
 
+#define SECONDS_TO_TIMER(seconds) ((seconds) * 100)
+
+#define OSTIME_TO_TIMER(osTime) ((osTime) * 64 / 3000 / 10000)
+#define OSTIME_TO_TIMER_ALT(osTime) ((osTime) / 10000 * 64 / 3000)
+
+// 1 centiSecond = 10 milliSeconds = 1/100 seconds
```

Comment:
I think you might as well write the words in the documentation in the usual way rather than camelCase, especially since they are single words. Actually I think I'd just prefer them written as a single word in general anyway.